configure

```diff
@@ -11837,6 +11837,60 @@ $as_echo "#define HAVE_LCHFLAGS 1" >>confdefs.h
 
 fi
 
+
+# on alpine, lchmod raises errno EOPNOTSUPP on symlinks
+{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for broken lchmod" >&5
+$as_echo_n "checking for broken lchmod... " >&6; }
+if ${ac_cv_have_lchmod+:} false; then :
+  $as_echo_n "(cached) " >&6
+else
+  if test "$cross_compiling" = yes; then :
+  ac_cv_have_lchmod=cross
+else
```

Comment:
Is this else for the inner if-block? Should it be indented one level, along with an extra level for the `cat` command below it?

Actually, It looks like there's a ton of nested stuff below that isn't indented -- it makes it hard to read, but I don't know if that is a style thing I'm not used to since I haven't worked specifically in `configure` files much...

Reply:
`configure` is entirely machine generated, the actual code is in `configure.am`